lib/renderer/api/desktop-capturer.js

```diff
@@ -1,46 +1,43 @@
-const ipcRenderer = require('electron').ipcRenderer
-const nativeImage = require('electron').nativeImage
+const {ipcRenderer, nativeImage} = require('electron')
 
-var nextId = 0
-var includes = [].includes
+const includes = [].includes
```

Comment:
What's the reason why we have to save a reference to the method?

Reply:
hmm, this was something i didn't really change, so i'm not sure the original thinking behind it. would it be better without it?

Reply:
> would it be better without it?

Yeah, cleaner code is better code.

There might be two reasons to save `includes` in a local var:
1) It is used not on `Array`, but on an Array-like object with nonexistent or redefined `includes()` method.
2) `Array.prototype.includes()` might be redefined and we want to make sure that we use an original method.